www/core/nightlies/next_major_list.xml

```diff
@@ -0,0 +1,7 @@
+<extensionset name="Joomla! Core Nightly Builds" description="Joomla! Core Next Major Nightly Builds">
+	<extension name="Joomla" element="joomla" type="file" version="5.0.0-dev" targetplatformversion="4.0" detailsurl="https://update.joomla.org/core/nightlies/next_minor_extension.xml" />
```

Comment:
@zero-24 Are we assuming users can update from 4.0 to 5.0 or should they go through 4.0.4 first?

Reply:
Adjusted the update server to go to 4.0.4 first.